tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -446,6 +452,14 @@ def do_testharness(self, protocol, url, timeout):
             done, rv = handler(result)
             if done:
                 break
+
+        # Attempt to cleanup any leftover windows, if allowed. This is
+        # preferable as it will blame the correct test if something goes wrong
+        # closing windows, but if the user wants to see the test results we
+        # have to leave the window(s) open.
+        if self.may_pause_after_test:
```

Comment:
My brain isn't parsing why `may_pause_after_test` is `False` in the case that we actually might pause after the test is done. I think this should be the other way around. But honestly I think I'd like the flag to just be `cleanup_after_test` and the booleans to be the same way around.

Reply:
Done.